test/e2e/storage/pd.go

```diff
@@ -188,6 +190,8 @@ var _ = utils.SIGDescribe("Pod Disks", func() {
 					ginkgo.By("deleting host0Pod") // delete this pod before creating next pod
 					framework.ExpectNoError(podClient.Delete(host0Pod.Name, podDelOpt), "Failed to delete host0Pod")
 					framework.Logf("deleted host0Pod %q", host0Pod.Name)
+					e2epod.WaitForPodToDisappear(cs, host0Pod.Namespace, host0Pod.Name, labels.Everything(), framework.Poll, framework.PodDeleteTimeout)
```

Comment:
Since we're doing both sync + wait for pod to disappear, then we shouldn't need to retry the read?

Reply:
yeah I was leaning on the cautious side but I think we should remove this.